python/dgl/distributed/graph_services.py

```diff
@@ -260,27 +259,38 @@ def sample_neighbors(g, nodes, fanout, edge_dir='in', prob=None, replace=False):
     Parameters
     ----------
     g : DistGraph
-        The distributed graph.
+        The distributed graph..
     nodes : tensor or dict
-        Node ids to sample neighbors from. If it's a dict, it should contain only
+        Node IDs to sample neighbors from. If it's a dict, it should contain only
         one key-value pair to make this API consistent with dgl.sampling.sample_neighbors.
     fanout : int
-        The number of sampled neighbors for each node.
+        The number of edges to be sampled for each node.
+
+        If -1 is given, all of the neighbors will be selected.
     edge_dir : str, optional
-        Edge direction ('in' or 'out'). If is 'in', sample from in edges. Otherwise,
-        sample from out edges.
+        Determines whether to sample inbound or outbound edges.
+
+        Can take either ``in`` for inbound edges or ``out`` for outbound edges.
     prob : str, optional
-        Feature name used as the probabilities associated with each neighbor of a node.
-        Its shape should be compatible with a scalar edge feature tensor.
+        Feature name used as the (unnormalized) probabilities associated with each
+        neighboring edge of a node.  The feature must have only one element for each
+        edge.
+
+        The features must be non-negative floats, and the sum of the features of
+        inbound/outbound edges for every node must be positive (though they don't have
+        to sum up to one).  Otherwise, the result will be undefined.
     replace : bool, optional
         If True, sample with replacement.
 
+        When sampling with replacement, the sampled subgraph could have parallel edges.
```

Comment:
parallel edges -> duplicated edges?

Reply:
i copied from sample_neighbors. we should use a consistent term for this kind of edges. @BarclayII